src/Stratis.Bitcoin.Features.ColdStaking/Controllers/ColdStakingWalletRPCController.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Extensions.Logging;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Wallet.Interfaces;
+
+namespace Stratis.Bitcoin.Features.ColdStaking.Controllers
+{
+    /// <summary> All functionality is in WalletRPCController, just inherit the functionality in this feature.</summary>
+    public class ColdStakingWalletRPCController : WalletRPCController
```

Comment:
Why do we need this? is the low level controller not registered by default?

Reply:
Check out #2986, it's because the RPC controller discovery is done at the feature level by assembly and the wallet RPC controller resides in another feature's assembly.

Reply:
ah right ok yes and we never actually register the wallet controller.